app/javascript/react/components/MovieShow.js

```diff
@@ -1,11 +1,25 @@
 import React from "react"
 
 const MovieShow = (props) => {
+  let poster
+  if(props.movie_poster){
+    if(props.movie_poster.url !== null){
+      poster = <img className="padded" src={props.movie_poster.url} alt="movie poster" />
+    }
+  }
```

Comment:
This is perfectly fine and not a requested change, but we now have access to optional chaining (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) which can help get around nested conditionals